frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeList/hooks/useValidateInputCodeListName.ts

```diff
@@ -0,0 +1,22 @@
+import { FileNameValidationResult } from '@studio/pure-functions';
+import { useTranslation } from 'react-i18next';
+
+export function useValidateInputCodeListName() {
+  const { t } = useTranslation();
+
+  type FileNameValidationErrorResult = Exclude<
+    FileNameValidationResult,
+    FileNameValidationResult.Valid | FileNameValidationResult.NoRegExMatch
+  >;
+
+  const errorMessages: Record<FileNameValidationErrorResult, string> = {
+    [FileNameValidationResult.FileNameIsEmpty]: t('validation_errors.required'),
+    [FileNameValidationResult.FileExists]: t('validation_errors.file_name_occupied'),
+  };
+
+  const getInvalidInputFileNameErrorMessage = (fileNameError: FileNameValidationResult) => {
+    return errorMessages[fileNameError];
+  };
+
+  return { getInvalidInputFileNameErrorMessage };
```

Comment:
Nå som denne bare returnerer én funksjon, trenger den vel ikke å være i et objekt?
```suggestion
  return (fileNameError: FileNameValidationResult) => errorMessages[fileNameError];
```